### ISSUE TITLE
Back end skills

### ISSUE THREAD

**Issue by mehul-m-prajapati** [Full-Stack-Development-Guide/technology_stack#2]
What do you gain?

TL;DR: Good cloud/backend engineering

- AWS, Azure, etc
- Architecting robust backends
- Designing and deploying cloud infrastructure
- Get great at Java, Go, NodeJS (and maybe Python)
- Lifecycle of APIs
- System designs and reviews
- Documentation

- Load balancing
- Fault-tolerance
- API Security
- Security and Data Privacy
- Authentication and Authorization
- Proxies and Reverse-Proxies
- All testing
- SRE
- Chaos and Load testing - Generated report provided
- and more

Why should you attend?

- Your services will be battle-tested and a public proof of your work will be generated that will demonstrate that you performed on a high quality bar and know how to maintain services reliably.

- If you are a student, do you want to walk into an interview and say “I know data structures”, OR “I have deployed, upgraded and managed backend services that can handle upto 2000 API requests per second while keeping costs low, here is the API test suite & reports.”
